source/mapping-rules/new-rules-to-mongodb.txt

```diff
@@ -54,6 +54,11 @@ Steps
    mapping rule, you can:
 
    - Change the collection name. 
+     
```

Comment:
I may be wrong about what this is referencing but the collection rename field is accessible "outside" of the configuration of any specific mapping 
![image](https://github.com/mongodb/docs-relational-migrator/assets/56749912/672eacf4-accd-475c-8a5b-5c3b4a0b070c)

Renaming the collection will only be possible if starting from the relational menu and adding a "New Document" mapping or clicking one of the MongoDB collections but not going into the "Add" menu.

Reply:
Agree. I actually think this should go in a different topic, alongside where we put the time series / collection type docs.

Reply:
Thank you, I have added a small task page in that section!